README.rst

```diff
@@ -182,9 +196,10 @@ installed on your system:
 
 - check
 - glib
+- golang - version 1.7 or above
```

Comment:
I think that we should also specify the minimum required version of all packages
```
glib - version 2.46.2 or above
....
```
or is enough with versions.txt ?